src/site/content/en/blog/vitals-tools-workflow/index.md

```diff
@@ -0,0 +1,507 @@
+at [g.co/chromeuxdash](http://g.co/chromeuxdash),
+based on the [Chrome UX Report](/chrome-ux-report/) (CrUX) data.
+CrUX is powered by key user experience metrics across the public web,
+aggregated from real Chrome users who have opted-in to syncing their browsing history
+(as such, it does not report the experience of all users of a website).
+
+{% Aside %}
+A website might not have enough eligible traffic
+to be included in the CrUX dataset to generate this report.
+You can capture real-time key user metrics by
+[integrating the web-vitals.js library](/vitals/#measure-core-web-vitals-in-javascript)
+or using other RUM solutions.
+This is a good practice even for websites with enough CrUX data as it allows monitoring on a larger user base.
+{% endAside %}
+
+The CrUX report provides information on:
+
+- **Overview of the site:**
+split by desktop and mobile for the key CWV Metrics
+(Largest Contentful Paint, Cumulative Layout shift, and First Input Delay)
+in addition to other metrics such as First Contentful Paint and Time To First Byte.
+- **Historical Trend by Metric type:**
+each page includes a chart showing distributions over time
+for each available monthly release of the CrUX report data.
+- **User Demographics:**
+these pages illustrate the distribution of page views across the entire origin
+for users in each demographic (devices and effective connection types).
+
+With this dashboard you can:
+
+- Get insights on overall website stability
+and performance per device type, and identify areas of improvement.
+- Understand how architectural changes or builds to the site
+have impacted user experience over time
+(overlaying your historical release data with the CrUX dashboard monthly results,
+to understand what caused regressions).
+- Narrow down on regression timelines, providing the focus for further investigation.
+
+To know more about how to use the CruX dashboard, read
+[Using the dashboard](/chrome-ux-report-data-studio-dashboard/#using-the-dashboard).
+
+Once you get the overall picture,
+the next step is to identify which pages or templates need the most attention.
+These origin-level reports do not provide this insight,
+but the Search Console does.
+
+### Identify poor performing pages using Google Search Console
+
+<figure class="w-figure">
+{% Img
+src="image/ZDZVuXt6QqfXtxkpXcPGfnygYjd2/w3CgACGvcnqjORHfWVNg.png",
+alt="A screenshot of Core Web Vitals charts.",
+width="800", height="639" %}
+  <figcaption class="w-figcaption">
+    Core Web Vitals report in the Search Console.
+  </figcaption>
+</figure>
+
+[The Core Web Vitals report in Search Console](https://support.google.com/webmasters/answer/9205520)
+provides a holistic site-wide view of your performance
+and helps identify the types of pages that need the most attention.
+Using this tool you can:
+
+- Identify the number of pages with a poor experience,
+need improvements, or a good experience,
+broken down by mobile and desktop views.
```

Comment:
```suggestion
- Identify the number of pages with a poor experience,
that need improvements, or a good experience,
broken down by mobile and desktop views.
```

Reply:
Because this is already published, I've skipped over several things I could have requested changes to. 

I have another problem with this passage, specifically the phrase 'a good experience'. Am I identifying pages that need a better user experience or am I identifying pages that are already good?